crates/hc_sandbox/src/zome_call.rs

```diff
@@ -275,7 +275,10 @@ pub async fn zome_call(zome_call: ZomeCall, admin_port: Option<u16>) -> anyhow::
         )
         .await?;
 
-    serde_json::to_writer(std::io::stdout(), &response.decode::<serde_json::Value>()?)?;
+    serde_json::to_writer(
+        std::io::stdout(),
+        &response.decode::<hc_serde_json::Value>()?,
```

Comment:
This change makes the test pass. I suppose this was the only missing part where byte arrays were returned and not deserialized correctly.